invenio_cli/helpers/process.py

```diff
@@ -52,7 +53,10 @@ def run_interactive(command, env=None, skippable=False):
             full_env[var] = val
 
     try:
-        response = run(command, check=True, env=full_env)
+        stdout = open(log_file, "a") if log_file else None
+        response = run(command, check=True, env=full_env, stdout=stdout, stderr=stdout)
+        if stdout:
+            stdout.close()
```

Comment:
major: This should be closed in a `finally`. Otherwise, if there is an exception the descriptor would still be open.

Reply:
fixed